CHANGES.rst

```diff
@@ -5,6 +5,13 @@
 1.7.1 (2022-09-07)
 ==================
 
+assign_wcs
+----------
+
+- Fixed a bug in ``assign_wcs`` due to which the step could crash due to
+  uncaught exception when SIP approximation fails to reach desired
+  accuracy. [#7036]
```

Comment:
This log should go under 1.7.2. You need to rebase if you don't see the 1.7.2 section.